modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -45,4 +45,19 @@ spec:
 ----
```

Comment:
This question is for the above lines that I can't comment on

I know this is for the other desscheduler thing, but did you have a separate PR to update all instances of `DevPreviewLongLifecycle` in this file (and any other places in the virt assembly) to `LongLifecycle`?